service/history/tasks/delete_visibility_task.go

```diff
@@ -83,6 +83,6 @@ func (t *DeleteExecutionVisibilityTask) GetType() enumsspb.TaskType {
 	return enumsspb.TASK_TYPE_VISIBILITY_DELETE_EXECUTION
 }
 
-func (t *DeleteExecutionVisibilityTask) GetCloseVisibilityTaskID() int64 {
+func (t *DeleteExecutionVisibilityTask) GetCloseExecutionVisibilityTaskID() int64 {
```

Comment:
Is this method used somewhere? I guess I am still confused why we need this method like I commented in an earlier PR.

Reply:
Removed